### PR TITLE
Added my name/GitHub link to the Contributors section on the About page.

### PR DESCRIPTION
**Pull request by CoderAJ7470**
## Related Issue

Resolves #

### Checklist

- [x] I have read and understand the **CODE OF CONDUCT** as outlined
- [x] I have read the **CONTRIBUTING** document
- [x] I have validating my HTML as outlined in the **CONTRIBUTING** document
- [x] My repository is up to date, and merge conflicts have been resolved
- [x] This pull request relates to an open issue and has been tagged above

### CODE REVIEW

File: docs/about.html [hunk 1]
```diff
@@ -102,6 +102,7 @@ <h2 id="organizers-volunteers">Contributors</h2>
                         <li><a href="https://github.com/stu7" target="_blank" title="Norman Haase" rel="external">Norman Haase</a></li>
                         <li><a href="https://github.com/Jason9199" target="_blank" title="Jason Carpenter" rel="external">Jason Carpenter</a></li>
                         <li><a href="https://github.com/brycejech" target="_blank" title="Bryce Jech" rel="external">Bryce Jech</a></li>
+						<li><a href="https://github.com/coderaj7470" target="_blank" title="Ameya Joshi" rel="external">Ameya Joshi</a></li>
```
Comment (mattstub) [inline docs/about.html]: could you remove the extra whitespace here @CoderAJ7470 and bring it in line with the other list items? thanks!

**Review by mattstub — CHANGES_REQUESTED** [review]
see whitespace change, let me know if you have any questions on making this

**Comment by CoderAJ7470** [comment]
Hello Matt,

I have deleted the whitespace that was appearing at the start of the link I added for my name/GitHub profile. It should be corrected now. :)

**Review by mattstub — APPROVED** [review]
approved, thanks @CoderAJ7470

**Pull request merged**